CodeFormatter.py

```diff
@@ -64,49 +64,22 @@ def plugin_loaded():
 
 class CodeFormatterCommand(sublime_plugin.TextCommand):
 
-    def run(self, edit, syntax=False, saving=False):
+    def run(self, edit, syntax=None, saving=None):
+        run_formatter(self.view, edit, syntax=syntax, saving=saving)
 
-        if self.view.is_scratch():
-            return show_error('File is scratch')
 
-        file_name = self.view.file_name()
+class CodeFormatterOpenTabsCommand(sublime_plugin.TextCommand):
```

Comment:
this could extend CodeFormatterCommand and call `parent.run` multiple times instead of separate function which both classes now use

(i am not sure if this will be better solution)

Reply:
I don't think this is a good idea. We would need to add the quiet parameter in run function of CodeFormatterCommand, which is (currently) only used by CodeFormatterOpenTabsCommand. And also, we are not duplicating any code that justifies such change.